integration-tests/cli/resource-deleter.it.js

```diff
@@ -0,0 +1,335 @@
+import fetch from 'node-fetch'
+import Promise from 'bluebird'
+import { createAuthMiddlewareForClientCredentialsFlow } from '@commercetools/sdk-middleware-auth'
+import { createClient } from '@commercetools/sdk-client'
+import { createRequestBuilder } from '@commercetools/api-request-builder'
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http'
+import ResourceDeleter from '@commercetools/resource-deleter'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'mz/child_process'
+import { version } from '@commercetools/resource-deleter/package.json'
+import * as resources from './helpers/resource-delete.data'
+import { createData, clearData } from './helpers/utils'
+
+const projectKey =
+  process.env.CI === 'true'
+    ? 'resource-deleter-int-test'
+    : process.env.npm_config_projectkey
+
+describe('Resource Deleter', () => {
+  let apiConfig
+  let resourceDeleter
+  let logger: {
```

Comment:
why not import this one from here? https://github.com/commercetools/nodejs/blob/9e598aaaed4ec29d232538cef1ed7bd189ee7136/packages/resource-deleter/src/utils/silent-logger.js